tensorflow/java/src/main/java/org/tensorflow/op/annotation/Operator.java

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+package org.tensorflow.op.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation used by classes that should be added to the Ops convenience API.
+ *
+ * <p>An annotation processor builds the {@code Ops} API by aggregating all classes annotated as
```

Comment:
Yes, in a future PR :-) I couldn't come up with a good way of documenting the intent of the annotation without referring to a yet-to-come processor.  Suggestions? I'll insert a TODO in the javadoc to make it clear it doesn't exist yet.